examples/admin-development/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "admin-development",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "@babel/core": "^7.16.7",
+    "@babel/plugin-proposal-class-properties": "7.16.7",
+    "@babel/plugin-proposal-decorators": "7.16.7",
+    "@babel/plugin-proposal-export-default-from": "7.16.7",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "7.16.7",
+    "@babel/plugin-proposal-object-rest-spread": "7.16.7",
+    "@babel/plugin-proposal-optional-chaining": "7.16.7",
+    "@babel/plugin-proposal-private-methods": "7.16.7",
+    "@babel/plugin-syntax-dynamic-import": "7.8.3",
+    "@babel/plugin-transform-arrow-functions": "7.16.7",
+    "@babel/plugin-transform-block-scoping": "7.16.7",
+    "@babel/plugin-transform-classes": "7.16.7",
+    "@babel/plugin-transform-destructuring": "7.17.7",
+    "@babel/plugin-transform-for-of": "7.16.7",
+    "@babel/plugin-transform-parameters": "7.16.7",
+    "@babel/plugin-transform-shorthand-properties": "7.16.7",
+    "@babel/plugin-transform-spread": "7.16.7",
+    "@babel/preset-env": "7.16.11",
+    "@babel/preset-typescript": "7.16.7",
+    "@storybook/addon-actions": "6.4.10",
+    "@storybook/addon-essentials": "6.4.10",
+    "@storybook/addon-interactions": "6.4.10",
+    "@storybook/addon-links": "6.4.10",
+    "@storybook/react": "^6.3.7",
+    "@storybook/testing-library": "^0.0.9",
+    "babel-loader": "^8.2.4",
+    "babel-plugin-macros": "3.1.0",
+    "babel-plugin-polyfill-corejs3": "0.5.2",
+    "core-js": "3.21.1"
+  },
+  "dependencies": {},
+  "scripts": {
+    "storybook": "start-storybook -p 6006",
```

Comment:
What do you think about running this on `6007`, so that we can run the one from the helper-plugin and this one at the same time?